google/cloud/bigtable/row_filters.py

```diff
@@ -35,6 +41,19 @@ class RowFilter(object):
         This class is a do-nothing base class for all row filters.
     """
 
+    def to_pb(self) -> data_v2_pb2.RowFilter:
```

Comment:
The original implementation of RowFilter had `to_pb` functions for each filter, but I don't think we'll need it for the v3 client. I'm leaning towards removing `to_pb` for simplicity (and in case we change proto formats in the future), but I'm curious if there are any reasons for keeping it around, since it exists and is well tested.

Reply:
I would suggest keeping it public, because it is already available in v2. We don't know whether users are relying on this or not. I think it will make it easier for the to upgrade to v3 if they have access to similar functions. Otherwise we need to start raising deprecation warning saying this will be removed in v3.